exploration-module/src/main/java/io/edpn/backend/exploration/adapter/persistence/MybatisSystemCoordinateRequestRepository.java

```diff
@@ -15,7 +15,12 @@
 
 public interface MybatisSystemCoordinateRequestRepository {
 
-    @Insert("INSERT INTO system_coordinate_data_request (requesting_module, system_name) VALUES (#{requestingModule}, #{systemName})")
+    @Insert({
+            "INSERT INTO system_coordinate_data_request (requesting_module, system_name)",
+            "VALUES (#{requestingModule}, #{systemName})",
+            "ON CONFLICT (requesting_module, system_name)",
+            "DO NOTHING"
```

Comment:
:heart_eyes:

Reply:
I haven't used MyBatis, but since the annotations already indicate the operation, there's no requirement to have a certain method name? In which case, perhaps change it to reflect the functionality, something to the effect of "insertIfNotExists()"?